app/assets/src/components/views/SampleUploadFlow/UploadMetadataStep.jsx

```diff
@@ -67,6 +70,13 @@ class UploadMetadataStep extends React.Component {
         issues: this.state.issues
       });
     }
+    logAnalyticsEvent("UploadMetadataStep_continue-button_clicked", {
```

Comment:
I would log "this.state.wasManual" as well, since there are two code paths in this function.

Also, this.setState is asynchronous, so this.state might not immediately update after you call it. `this.state.issues` here might not reflect the `setState({ issues })` call on line 58. You'll want to use `result.issues`.

Reply:
Good catch! I normally watch out for `setState` but I missed this one. 

I'll log `wasManual` as well.